### PR TITLE
Taking a little bit of ego out of the codebase

### PR DESCRIPTION
**Pull request by RichardPK**
**Overall change:**
- Removing the name 'Drew' from the MAP fixtures and updating all relevant snapshots.

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] I have assigned this PR to the Simorgh project

**Testing:**

- [x] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)

### CODE REVIEW

File: data/pidgin/cpsAssets/23248703.json [hunk 1]
```diff
@@ -571,7 +571,7 @@
   },
   "promo": {
     "headlines": {
-      "headline": "Simorgh: Media Pod Build First CPS Media Asset Page in Simorgh with the Help of Drew & < >",
+      "headline": "Simorgh: Media Pod Build First CPS Media Asset Page in Simorgh & < >",
```
Comment (karinathomasbbc) [inline data/pidgin/cpsAssets/23248703.json:574]: Hey, I know this is a draft (sorry for 👀 ) - why don't we replace the entire contents of this one with what's on the test CPS (https://www.test.bbc.com/pidgin/23248703.json)

File: src/app/pages/MediaAssetPage/fixtureData.json [hunk 2]
```diff
@@ -1860,7 +1860,7 @@
   },
   "promo": {
     "headlines": {
-      "headline": "Simorgh: Media Pod Build First CPS Media Asset Page in Simorgh with the Help of Drew & < >",
+      "headline": "Simorgh: Media Pod Build First CPS Media Asset Page in Simorgh & < >",
```
Comment (thekp) [inline src/app/pages/MediaAssetPage/fixtureData.json:1863]:
```suggestion
      "headline": "Simorgh: Media Pod Build First CPS Media Asset Page in Simorgh, bye bye Khoa & < >",
```

Just a small suggestion :trollface:

Comment (karinathomasbbc) [reply src/app/pages/MediaAssetPage/fixtureData.json:1863]: 😂 and also 😭

File: data/pidgin/cpsAssets/23248703.json (hunk 1 above)
Comment (RichardPK) [reply data/pidgin/cpsAssets/23248703.json:574]: That probably makes a lot more sense but it's much less of a _statement_ piece than just removing the name

File: src/app/pages/MediaAssetPage/fixtureData.json (hunk 2 above)
Comment (RichardPK) [reply src/app/pages/MediaAssetPage/fixtureData.json:1863]: :laughing-but-crying-face:

Comment (thekp) [reply src/app/pages/MediaAssetPage/fixtureData.json:1863]: 😆 Cheers @ryanmccombe

File: src/app/pages/MediaAssetPage/fixtureData.json [hunk 3]
```diff
@@ -1860,7 +1860,7 @@
   },
   "promo": {
     "headlines": {
-      "headline": "Simorgh: Media Pod Build First CPS Media Asset Page in Simorgh with the Help of Drew & < >",
+      "headline": "Simorgh: Media Pod Build First CPS Media Asset Page in Simorgh, bye bye Khoa & < >",
```
Comment (thekp) [inline src/app/pages/MediaAssetPage/fixtureData.json]:
```suggestion
      "headline": "Simorgh: Media Pod Build First CPS Media Asset Page in Simorgh & < >",
```
trolling aside 😅 thanks for the love my Scottish friends 💪

**Review by 12 — APPROVED** [review]

**Review by karinathomasbbc — APPROVED** [review]

**Review by DenisHdz — APPROVED** [review]

**Review by thekp — APPROVED** [review]

**Review by DemonusPC — APPROVED** [review]

**Review by thekp — APPROVED** [review]

**Review by dr3 — APPROVED** [review]

**Comment by RichardPK** [comment]
I think this puppy is ready to fly

**Pull request merged**